### ISSUE TITLE
Single letter type arguments followed by `::` cause problems on Windows

### ISSUE THREAD

**Issue by noisegul** [noisegul/perl6-p6doc#21]
When the `p6doc` command is invoked on Windows CMD, single letters followed by `:` will be converted to drive paths, so it seems.

Thus, executing `p6doc X::IO`, `X::IO` will be given to `MAIN` as `X:/:IO`, this behaviour can be observed in the original `p6doc`.

This can be avoided by putting the whole argument between "" on Windows, however since this only happens if the single letter Type comes first, this should be easy to catch and handle.

**Comment by JJ** [comment]
How's that fixed in Perl 5's pod?

**Comment by noisegul** [comment]
Can you elaborate?

**Comment by JJ** [comment]
perldoc is part of the Perl distribution. They must have done something about it, not clear what.

**Comment by JJ** [comment]
(not clear because I don't use Windows and haven't checked out the code)

**Comment by noisegul** [comment]
It turns out that my statement was wrong, after taking another look at this I was no longer able to reproduce this on `cmd.exe` and I do not know why, seems like I missed something else about `cmd.exe` usage.

Single letters followed by `::` are handed to `MAIN` correctly again, when I opened this issue it happened with original `p6doc` and any test program I created to confirm it.

I am closing this issue for now, will reopen it should it resurface.
